expression/builtin_string_vec_test.go

```diff
@@ -218,6 +218,10 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 	ast.InsertFunc: {
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString, types.ETInt, types.ETInt, types.ETString}, geners: []dataGenerator{&randLenStrGener{10, 20}, &rangeInt64Gener{-10, 20}, &rangeInt64Gener{0, 100}, &randLenStrGener{0, 10}}},
 	},
+	ast.Format: {
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETDecimal, types.ETInt}, geners: []dataGenerator{&rangeDecimalGener{}, &rangeInt64Gener{2, 5}}},
```

Comment:
We can use `rangeInt64Gener{0, 30}` to cover more cases.

Reply:
I will fix it.